src/envoy/mixer/http_filter.cc

```diff
@@ -257,6 +258,37 @@ class CheckData : public HttpCheckData,
     }
     return false;
   }
+
+  bool GetJWTPayload(
+      std::map<std::string, std::string>* payload) const override {
+    if (payload == nullptr) {
+      return false;
+    }
+    const std::string kSecIstioAuthUserinfo("sec-istio-auth-userinfo");
```

Comment:
call this function https://github.com/istio/proxy/blob/master/src/envoy/auth/http_filter.cc#L31
to get the key.  Don't define another one here